lib/charms/lte_core_interface/v0/lte_core_interface.py

```diff
@@ -237,6 +236,7 @@ def set_lte_core_information(self, mme_ipv4_address: str) -> None:
         if not self.model.get_relation(self.relationship_name):
             raise RuntimeError(f"Relation {self.relationship_name} not created yet.")
         if not self._mme_ipv4_address_is_valid(mme_ipv4_address):
+            self.charm.unit.status = BlockedStatus("Invalid MME IPv4 address.")
```

Comment:
The interface shouldn't be capable of putting charms in different statuses. That's up to the charm author. Here we can raise an error to be caught by the charm.

Reply:
Fixed.